akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sharding/MultiNodeClusterShardingSpecSpec.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding
+
+import scala.concurrent.duration._
+
+import akka.actor._
+import akka.cluster.MemberStatus
+import akka.testkit._
+import com.typesafe.config.ConfigFactory
+
+abstract class MultiNodeClusterShardingSpecConfig(mode: String, rememberEntities: Boolean)
+    extends MultiNodeClusterShardingConfig(mode, rememberEntities) {
+
+  val first = role("first")
+  val second = role("second")
+  val third = role("third")
+  val fourth = role("fourth")
+
+  nodeConfig(third)(ConfigFactory.parseString(s"""
+    akka.cluster.sharding.distributed-data.durable.lmdb.dir = $targetDir/sharding-third
+    """))
+
+}
+
+object PersistentClusterShardingRememberEntitiesEnabledConfig
+    extends MultiNodeClusterShardingSpecConfig(ClusterShardingSettings.StateStoreModePersistence, true)
+
+class PersistentClusterShardingMultiNodeSpecMultiJvmNode1
```

Comment:
I don't think these can be defined here, must be in the specific tests with unique naming I think

Reply:
Used in test of base test, the way we test test kits, just the *SpecSpec

Reply:
and also, maybe we don't need a SpecSpec for this and instead test it via real tests (it's just an internal utility, and adding more run time to build might not be worth it)